backend/apps/startups/apps.py

```diff
@@ -4,3 +4,4 @@
 class StartupsConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'apps.startups'
+    label = 'startups'
```

Comment:
AppConfig.name is still 'apps.startups' but settings use 'backend.apps.startups', causing app-loading mismatch. Set name to the full import path to match INSTALLED_APPS.
```
class StartupsConfig(AppConfig):
    default_auto_field = 'django.db.models.BigAutoField'
    name = 'backend.apps.startups'
    label = 'startups'
```